src/storage/src/table/streaming_table/state_table.rs

```diff
@@ -655,16 +670,20 @@ impl<S: StateStore> StateTable<S> {
 impl<S: StateStore> StateTable<S> {
     /// This function scans rows from the relational table.
     pub async fn iter(&self) -> StorageResult<RowStream<'_, S>> {
-        self.iter_with_pk_prefix(Row::empty()).await
+        self.iter_with_pk_prefix(Row::empty(), false).await
```

Comment:
I feel like we can remove the `use_curr_epoch` flag. Will there be a problem if we always use the current epoch for reads?